src/context/editorReducers.js

```diff
@@ -13,33 +13,97 @@
 export const UPDATE_OFFLINE_TD = 'UPDATE_OFFLINE_TD';
 export const UPDATE_IS_MODFIED = 'UPDATE_IS_MODFIED';
 export const SET_FILE_HANDLE = 'SET_FILE_HANDLE';
+export const REMOVE_FORM_FROM_TD = 'REMOVE_FORM_FROM_TD';
+export const ADD_FORM_FROM_TD = 'ADD_FORM_FROM_TD';
```

Comment:
With ADD_FORM_FROM_TD I think ADD_FORM_TO_TD is meant.

Reply:
Thanks for that hint